File: second_semester/POO/Clase/Class.py
class Persona:  #nombre
    def __init__(self, nombre, edad): #Atributos
        self.nombre = nombre
        self.edad = edad

    def saludar(self): #metodo
        return f"Hola, me llamo {self.nombre} y tengo {self.edad} años."

persona1 = Persona("Juan", 30) #instasias
mensaje = persona1.saludar()
print(mensaje)

class Arbol:
    pass

File: second_semester/POO/Clase/Class.js
class Persona { //nombre
    constructor(nombre, edad) { //Atributos
        this.nombre = nombre;
        this.edad = edad;
    }
    saludar() {
        return `Hola, me llamo ${this.nombre} y tengo ${this.edad} años.`;
    }
}
// Uso de la clase
const persona1 = new Persona("Juan", 30);
console.log(persona1.saludar()); 